addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java

```diff
@@ -148,6 +162,21 @@ public NAStationDataBody getStationsDataBody(String equipmentId) {
         return null;
     }
 
+    public NAHealthyHomeCoachDataBody getHomecoachDataBody(String equipmentId) {
+        if (getHomeCoachApi() != null) {
+            try {
+                NAHealthyHomeCoachDataBody data = getHomeCoachApi().gethomecoachsdata(equipmentId).getBody();
+                updateStatus(ThingStatus.ONLINE);
+                return data;
+            } catch (Exception e) {
```

Comment:
Do you really need to catch the Exception-class or are there more specific exceptions you could catch like the IOException or maybe only RuntimeException's (which is not that good either). The current code and also catching RuntimeException's might 'hide' programming errors like NullPointerExceptions occurring.

https://www.google.nl/search?q=java%20why%20not%20to%20catch%20exception

Reply:
Will fix